.github/workflows/wheels.yml

```diff
@@ -19,19 +20,19 @@ jobs:
     strategy:
       matrix:
         # os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ ubuntu-latest ]
         arch: [x86_64, aarch64]
-        os: [ubuntu-latest]
 
     steps:
       - uses: actions/checkout@v2
 
       - uses: actions/setup-python@v2
         name: Install Python
         with:
-          python-version: '3.7'
+          python-version: '3.9'
 
       - uses: docker/setup-qemu-action@v1
-        if:  ${{matrix.arch}} == 'aarch64'
+        if:  startsWith(matrix.os, 'ubuntu')
```

Comment:
Probably the two conditions should go in "and", but it seems not to hurt so no problem.